docs/howto/operate/grafana.md

```diff
@@ -39,8 +43,40 @@ grafana:
       - secretName: grafana-tls
         hosts:
           - <grafana-domain>
+
+prometheus:
+  server:
+    ingress:
+      enabled: true
+      hosts:
+        - <prometheus-domain>
+      tls:
+        - secretName: prometheus-tls
+          hosts:
+            - <prometheus-domain>
+```
+
+#### Create a `enc-support.secret.yaml` file
+
+Only 2i2c staff + our centralized grafana should be able to access the
+prometheus data on a cluster from outside the cluster. The [basic auth](https://kubernetes.github.io/ingress-nginx/examples/auth/basic/)
+feature of nginx-ingress is used to restrict this. A `enc-support.seecret.yaml`
```

Comment:
I think it will be critical that this is named `enc-support.secret.values.yaml` --- that it includes `values.yaml` in the end to be compatible with https://github.com/2i2c-org/infrastructure/pull/1093/files#diff-6d3cb43608c85f4e234a8decb5a6dc2f7f62e5606891b377941a201fd8222d25R77.

```suggestion
feature of nginx-ingress is used to restrict this. A `enc-support.secret.values.yaml`
```

Reply:
I thought the secret.yaml is an alternative to .values.yaml, specifying what they are. Why have both?

Reply:
Either way, happy to just do this for now and move the conversation about conventions elsewhere :)

Reply:
All values files can be secret, but not all secrets need to be values files. So I think we should adopt a `.secret.values.yaml` convention to indicate that this is a secret values file. See the notes box in the docs here: https://infrastructure.2i2c.org/en/latest/topic/config.html#id1